### ISSUE TITLE
Abandonware?

### ISSUE THREAD

**Issue by myklemykle** [mikeloomisgg/cppack#22]
I looked at 5 different C++ messagepack implementations, and I really like how this one looks. I read the developer's [blog post](https://mikeloomisgg.github.io/2019-07-02-making-a-serialization-library/) and generally agree with his decisions. 

But there's been no activity on this repo in years. Several pull requests are stacked up & it looks like people would like to see it grow. What's the story? The developer is still (very slightly) active on GitHub so I hope he sees this message. 

Is this project still viable? Could it be handed off to another maintainer? Or is there something about it that fundamentally prevents the known issues from being fixed?  Or is MessagePack just obsolete in 2024 for some reason?

-m-

**Comment by jthompsonic** [comment]
I agree that the project should still be supported.  Hopefully the maintainer will kick back in and provide some kind of support or leadership of the project, or find another competent person to continue the project. Otherwise at some point the project can be forked with more support.
